BookingsAPI/Bookings.API/Controllers/ParticipantsController.cs

```diff
@@ -358,6 +358,29 @@ public IActionResult UpdateSuitabilityAnswers(Guid hearingId, Guid participantId
                 return BadRequest(ModelState);
             }
 
```

Comment:
or prticipantId ==Guid.Empty  should we add to bad reqest?

Reply:
Fixed